app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TweetsController < ApplicationController
+
+
+    get '/' do
+        erb :index
+    end
+    
+    get '/tweets' do
+        if logged_in?
+            erb :'tweets/tweets'   
+        else
+            redirect '/login'
+        end
+    end
+    
+    get '/tweets/new' do
+        if logged_in?
+            erb :"tweets/create_tweet"
+        else
+            redirect '/login'
+        end
+    end
+    
+    get '/tweets/:id' do
+        if logged_in?
+            @tweet = Tweet.find_by_id(params["id"])
+            erb :"tweets/show_tweet"
+        else
+            redirect "/login"
+        end
+    end
+    
+    get '/tweets/:id/edit' do
+        if logged_in?
+            @tweet = Tweet.find_by_id(params["id"])
```

Comment:
It's good practice to verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?